.github/workflows/test.yml

```diff
@@ -63,36 +69,38 @@ jobs:
       # For more: https://nodejs.org/en/download/releases/
       # (We also use this env var for making sure a step runs once for the current node version)
       IS_DEFAULT_NPM: ${{ (matrix.node == 18 && matrix.npm == 10) || (matrix.node == 20 && matrix.npm == 10) || (matrix.node == 22 && matrix.npm == 10) }}
-      # The current recommended version for Managed Runtime:
-      # https://developer.salesforce.com/docs/commerce/pwa-kit-managed-runtime/guide/upgrade-node-version.html
       IS_MRT_NODE: ${{ matrix.node == 22 && matrix.npm == 10 }}
+      PROJECT_DIR: generated-retail-react-app-test-project
     steps:
       - name: Checkout
         uses: actions/checkout@v4
 
-      - name: Setup Node
-        uses: actions/setup-node@v4
+      - name: Setup Ubuntu Machine
+        uses: "./.github/actions/setup_ubuntu"
         with:
-          node-version: ${{ matrix.node }}
-          cache: npm
+          node: ${{ matrix.node }}
+          npm: ${{ matrix.npm }}
+          update_npm: ${{ env.IS_DEFAULT_NPM == 'false' }}
 
-      - name: Update NPM version
-        if: env.IS_DEFAULT_NPM == 'false'
+      - name: Generate retail-react-app-test-project
+        if: env.IS_DEFAULT_NPM == 'true'
         run: |-
-          npm install -g npm@${{ matrix.npm }}
+          node packages/pwa-kit-create-app/scripts/create-mobify-app-dev.js --outputDir ${{ env.PROJECT_DIR }}
+        env:
+          GENERATOR_PRESET: retail-react-app-test-project
```

Comment:
Since retail-react-app no longer exists, we generate a project with this preset.